docs/data/sdks/android/index.md

```diff
@@ -553,7 +553,7 @@ You can also use `logEventWithGroups` to set event-level groups. This means that
     JSONObject eventProperties = new JSONObject().put("key", "value");
     JSONObject groups = new JSONObject().put("orgId", 10);
 
-    Amplitude.getInstance().logEvent("initialize_game", eventProperties, groups);
+    Amplitude.getInstance().logEventWithGroups("initialize_game", eventProperties, groups);
```

Comment:
logEventWithGroups not exist in android. For android, we are using the same name but with different parameters. 
<img width="612" alt="Screen Shot 2023-05-12 at 3 03 43 PM" src="https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/assets/13028329/4b233ec2-461b-4a8a-95b0-f4d2f0d22830">

Reply:
fixed and correct the description above